rust/noosphere/src/ffi/context.rs

```diff
@@ -528,3 +528,28 @@ pub fn ns_sphere_file_version_get(
             .map_err(|error: InvalidNulTerminator<String>| anyhow!(error).into())
     })
 }
+
+#[ffi_export]
+/// @memberof ns_sphere_file_t
```

Comment:
Sorry, almost missed this: should be changed to `ns_sphere_t`

Reply:
Nice catch!